### PR TITLE
Feature/srm 562 fix console errors

### PR DESCRIPTION
**Pull request by mayurngondhkar**
**Jira** https://digital-vic.atlassian.net/browse/SRM-562
# Changes

1.  Add `cdn.jsdelivr.net` to `script-src`, `font-src`, and `style-src` in `seckit`
2. Add `data:` to `font-src` in `seckit`

### CODE REVIEW

File: config/optional/seckit.settings.yml [hunk 1]
```diff
@@ -3,21 +3,21 @@ seckit_xss:
     checkbox: true
     report-only: false
     default-src: '''self'''
-    script-src: '''self'' ''unsafe-inline'' svc.webspellchecker.net'
+    script-src: '''self'' ''unsafe-inline'' svc.webspellchecker.net   cdn.jsdelivr.net'
```
Comment (MdNadimHossain) [inline config/optional/seckit.settings.yml:6]: spacing is wrong

File: config/optional/seckit.settings.yml [hunk 2]
```diff
@@ -3,21 +3,21 @@ seckit_xss:
...
     object-src: ''
     img-src: '''self'' data: *.vic.gov.au *.amazee.io tracking.monsido.com *.google-analytics.com *.g.doubleclick.net www.google.com *.google.com *.google.com.au api.mapbox.com svc.webspellchecker.net'
     media-src: ''
     frame-src: '''self'' batchgeo.com *.vimeo.com *.youtube.com livestream.com'
     child-src: ''
-    font-src: '''self'' svc.webspellchecker.net'
+    font-src: '''self'' svc.webspellchecker.net  cdn.jsdelivr.net data:'
```
Comment (MdNadimHossain) [inline config/optional/seckit.settings.yml:12]: spacing is wrong

File: config/optional/seckit.settings.yml [hunk 3]
```diff
@@ -3,21 +3,21 @@ seckit_xss:
...
     connect-src: '''self'' svc.webspellchecker.net'
     report-uri: /report-csp-violation
     policy-uri: ''
     vendor-prefix:
       x: false
       webkit: false
     upgrade-req: false
-    style-src: '''self''  ''unsafe-inline'' svc.webspellchecker.net'
+    style-src: '''self''  ''unsafe-inline'' svc.webspellchecker.net  cdn.jsdelivr.net'
```
Comment (MdNadimHossain) [inline config/optional/seckit.settings.yml:20]: spacing is wrong

**Review by MdNadimHossain — CHANGES_REQUESTED** [review]
small change, spacings are wrong

File: config/optional/seckit.settings.yml (hunk 3 above)
Comment (mayurngondhkar) [reply config/optional/seckit.settings.yml]: resolved

File: config/optional/seckit.settings.yml (hunk 2 above)
Comment (mayurngondhkar) [reply config/optional/seckit.settings.yml]: resolved

File: config/optional/seckit.settings.yml (hunk 1 above)
Comment (mayurngondhkar) [reply config/optional/seckit.settings.yml]: resolved

**Pull request merged**